vita3k/modules/SceGxm/SceGxm.cpp

```diff
@@ -1943,8 +1946,29 @@ EXPORT(int, sceGxmTextureInitLinear, SceGxmTexture *texture, Ptr<const void> dat
     return result;
 }
 
-EXPORT(int, sceGxmTextureInitLinearStrided) {
-    return UNIMPLEMENTED();
+EXPORT(int, sceGxmTextureInitLinearStrided, SceGxmTexture *texture, Ptr<const void> data, SceGxmTextureFormat texFormat, uint32_t width, uint32_t height, uint32_t byteStride) {
+    if (width > 4096 || height > 4096)
+        return RET_ERROR(SCE_GXM_ERROR_INVALID_VALUE);
+    else if (!data)
+        return RET_ERROR(SCE_GXM_ERROR_INVALID_ALIGNMENT);
+    else if (!texture)
+        return RET_ERROR(SCE_GXM_ERROR_INVALID_POINTER);
+
+    texture->mip_filter = ((byteStride >> 2) - 1) & 1;
```

Comment:
Make a temporary variable for this

```suggestion
    const std::uint32_t stride_compressed = (byteStride >> 2) - 1;
    texture->mip_filter = stride_compressed  & 1;
```

Same with other lines